doc/releases/changelog-0.25.0.md

```diff
@@ -175,232 +174,202 @@
-  includes:
-
   - You can now add scalars to Operators, where the interpretation is that the 
-  scalar is a properly-sized identity matrix. 
-
-
-  ```pycon
-  >>> sum_op = 5 + qml.CNOT([0,1])
-  >>> sum_op.matrix()
-  array([[5., 1.],
-         [1., 5.]])
-  ```
-
-* A `SProd` symbolic class is added that allows users to represent the scalar product
-of operators.
-
-  We can get the matrix, eigenvalues, terms, diagonalizing gates and more.
-
-  ```pycon
-  >>> sprod_op = qml.s_prod(2.0, qml.PauliX(0))
-  >>> sprod_op
-  2.0*(PauliX(wires=[0]))
-  >>> sprod_op.matrix()
-  array([[ 0., 2.],
-         [ 2., 0.]])
-  >>> sprod_op.terms()
-  ([2.0], [PauliX(wires=[0])])
-  ```
-
-  The `sprod_op` can also be used inside a `qnode` as an observable.
-  If the circuit is parameterized, then we can also differentiate through the observable.
-
-  ```python
-  dev = qml.device("default.qubit", wires=1)
-
-  @qml.qnode(dev, grad_method="best")
-  def circuit(scalar, theta):
-        qml.RX(theta, wires=0)
-        return qml.expval(qml.s_prod(scalar, qml.Hadamard(wires=0)))
-  ```
-
-  ```pycon
-  >>> scalar, theta = (1.2, 3.4)
-  >>> qml.grad(circuit, argnum=[0,1])(scalar, theta)
-  (array(-0.68362956), array(0.21683382))
-  ```
-
-* Adds the `Controlled` symbolic operator to represent a controlled version of any
-  operation.
+    scalar is a properly-sized identity matrix;
+
+    ```pycon
+    >>> sum_op = 5 + qml.PauliX(0)
+    >>> sum_op.matrix()
+    array([[5., 1.],
+           [1., 5.]])
+    ```
+    
+  - The `+` and `-` operators can be used to combine all Pennylane Operators:
+
+    ```pycon
+    >>> sum_op = qml.RX(phi=1.23, wires=0) + qml.RZ(phi=3.14, wires=0) - qml.RY(phi=0.12, wires=0)
+    >>> sum_op
+    RX(1.23, wires=[0]) + RZ(3.14, wires=[0]) + -1*(RY(0.12, wires=[0]))
+    >>> qml.matrix(sum_op)
+    array([[-0.18063077-0.99999968j,  0.05996401-0.57695852j],
+           [-0.05996401-0.57695852j, -0.18063077+0.99999968j]])
+    ```
+    Note that the behavior of `+` with *Observables* is different; it still creates 
+    a Hamiltonian.
+  
+  - The `*` and `@` operators can be used to scale and compose all PennyLane Operators.
+  
+    ```pycon
+    >>> prod_op = 2*qml.RX(1, wires=0) @ qml.RY(2, wires=0)
+    >>> prod_op
+    2*(RX(1, wires=[0])) @ RY(2, wires=[0])
+    >>> qml.matrix(prod_op)
+    array([[ 0.94831976-0.80684536j, -1.47692053-0.51806945j],
+           [ 1.47692053-0.51806945j,  0.94831976+0.80684536j]])
+     ```
+  
+  - The `**` operator can be used to raise PennyLane Operators to a power.
+  
+    ```pycon       
+    >>> exp_op = qml.RZ(1.0, wires=0) ** 2
+    >>> exp_op
+    RZ**2(1.0, wires=[0])
+    >>> qml.matrix(exp_op)
+    array([[0.54030231-0.84147098j, 0.        +0.j        ],
+           [0.        +0.j        , 0.54030231+0.84147098j]])
+    ```
+
+* A new class called `Controlled` is available in `qml.ops.op_math` to help represent 
+  a controlled version of any operation. 
```

Comment:
Good point. Feel like a degenerate feature to `qml.ctrl`

Reply:
`Controlled` is going to be a vast improvement over `ControlledOperation`, but it's a work-in-progress.

I'm currently working on adding basic decomposition functionality to `Controlled`, see #2938.  Once that PR is merged, I can have `ctrl` use `Controlled` instead of `ControlledOperation`.  Then we will deprecate `ControlledOperation`.

`ControlledOperation` just defines an `expand` and `adjoint`.

`Controlled` has:
* `label`
* `matrix`
* `sparse_matrix`
* `eigvals`
* `diagonalizing_gates`
* `generator`
* `adjoint`
* `pow`
* `simplify`
* `inverse` behavior
* `basis`
* `parameter_frequencies`
* `grad_method`
* Additional properties inheriting from `SymbolicOp`

Reply:
@albi3ro @josh146 see update to this section. Is this better?

Reply:
Nice, this sounds like a good plan @albi3ro! In this case, I wonder if we should describe and mention this `Controlled` option so visibly in the changelog for this release - it might be better to do so next release.

In which case, we could just shorten the entry. What do you think?

Reply:
@isaacdevlugt oops, I replied without refreshing the page, so didn't see your comment

Reply:
Still need clarification on this entry. I'm okay with moving to the improvements section as Christina mentioned 😄